### ISSUE TITLE
There is a problem with your tablet driver, please reboot your system

### ISSUE THREAD

**Issue by wilttd** [thenickdude/wacom-driver-fix#55]
removed all bamboo drivers and programs

installed:
My fixed Bamboo driver (v5.3.7-6) supports these tablets:

for tablet/mac:
CTH-661/s - Bamboo Fun
iMac Late 2013, macOS Catalina

The patched driver does not crash the OS while booting, works with your fingers. but not with the pen.

in the Wacom app in the system preferences i get the error:
There is a problem with your tablet driver, please reboot your system

**Comment by wilttd** [comment]
error before the there is a problem with your tablet driver is there is no tablet connected to your computer

**Comment by ellejaysee** [comment]
Same issue here. I removed the security & privacy items you listed but still see "PenTabletSpringboard". I'm not a coder, but an designer who is trying to figure out how to use this tablet again. Never had issues before. Your help is appreciated.
<img width="658" alt="Screen Shot 2020-12-05 at 1 36 05 PM" src="https://user-images.githubusercontent.com/75541676/101260575-0b5aea00-36ff-11eb-8a47-c63623760a9f.png">

**Comment by thenickdude** [comment]
@ellejaysee "PenTabletSpringboard" isn't a feature of my Bamboo driver, do you have another driver installed or is your tablet not a Bamboo tablet? In any case, don't untick that entry, use the minus button to remove it completely as the readme instructs.

**Comment by thenickdude** [comment]
@wilttd did you follow the instructions to remove all the tablet permissions from Input Monitoring and Accessibility and reboot?

**Comment by ellejaysee** [comment]
@thenickdude  ok so I just deleted that item and restarted. I see this when I reboot. Do I need to do anything with this? 
Regarding @wilttd's question, I deleted the tablet permissions from Input Monitoring and Accessibility sections and encountered the same thing as he. Hoping this is an easy fix!
<img width="480" alt="Screen Shot 2020-12-05 at 4 36 29 PM" src="https://user-images.githubusercontent.com/75541676/101265895-5a614900-3718-11eb-98d9-5d9f3e9255bf.png">

**Comment by thenickdude** [comment]
Please follow the instructions in the readme, you're at this section now:

https://github.com/thenickdude/wacom-driver-fix/blob/master/Readme.md#fix-up-the-tablet-permissions

**Comment by ellejaysee** [comment]
That worked!!!! Thank you!!

**Comment by wilttd** [comment]
Yes i did remove all permissions before installing the patched driver, even did the command line removal but no success.

**Comment by a2nl2e** [comment]
I was able to use the patch for about 10 days and then it stopped. I'm running catalina Intous3. I've uninstalled and reinstalled, but I can seem to drag springboard into Input Monitoring. Help please !
